tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -0,0 +1,107 @@
+from unittest.mock import Mock
+import dataactbroker.handlers.accountHandler
+import json
+from dataactcore.utils.jsonResponse import JsonResponse
+from dataactcore.utils.statusCode import StatusCode
+from tests.unit.dataactbroker.utils import insert_permissions, insert_user_statuses
+
+
+def test_max_login_success(database, monkeypatch):
+    insert_permissions(database)
+    insert_user_statuses(database)
+
+    ah = dataactbroker.handlers.accountHandler.AccountHandler(Mock())
+
+    mock_dict = Mock()
+    mock_dict.return_value.safeDictionary.side_effect = {'ticket': '', 'service': ''}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'RequestDictionary', mock_dict)
+
+    max_dict= {'cas:serviceResponse': {}}
+    monkeypatch.setattr(ah, 'get_max_dict', Mock(return_value=max_dict))
+    config = {'parent_group': 'parent-group'}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'CONFIG_BROKER', config)
+    max_dict = {
+        'cas:serviceResponse':
+            {
+                'cas:authenticationSuccess':
+                    {
+                        'cas:attributes':
+                            {
+                                'maxAttribute:Email-Address': 'test-user@email.com',
+                                'maxAttribute:GroupList': 'parent-group,parent-group-CGAC_SYS',
+                                'maxAttribute:First-Name': 'test',
+                                'maxAttribute:Middle-Name': '',
+                                'maxAttribute:Last-Name': 'user'
+                            }
+                    }
+            }
+    }
+    monkeypatch.setattr(ah, 'get_max_dict', Mock(return_value=max_dict))
+
+    # If it gets to this point, that means the user was in all the right groups aka successful login
+    monkeypatch.setattr(ah, 'create_session_and_response',
+                        Mock(return_value=JsonResponse.create(StatusCode.OK, {"message": "Login successful"})))
+    json_response = ah.max_login(Mock())
+
+    print(json.loads(json_response.get_data().decode("utf-8")))
+    assert "Login successful" == json.loads(json_response.get_data().decode("utf-8"))['message']
+
+
+def test_max_login_failure(monkeypatch):
+    ah = dataactbroker.handlers.accountHandler.AccountHandler(Mock())
+
+    mock_dict = Mock()
+    mock_dict.return_value.safeDictionary.side_effect = {'ticket': '', 'service': ''}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'RequestDictionary', mock_dict)
```

Comment:
Mock is necessary to mock the RequestDictionary class. Since we'll be revisiting that in the future, I didn't worry too much about it during testing.